### PR TITLE
[LC-892] Update third-party libraries

### PR DESCRIPTION
**Pull request by metalg0su**
(coupled: https://github.com/icon-project/icon-rpc-server/pull/129)

# Goal
- Update third-party libraries to the latest one (2020-03-02)

## Target Libraries
|Package         |Version|Latest |
|----------------|-------|-------|
|aio-pika        |3.0.1  |6.5.2 (REMOVED)|
|coloredlogs     |10.0   |14.0   |
|cryptography    |2.3.1  |2.8    |
|fluent-logger   |0.9.3  |0.9.5  |
|grpcio          |1.25.0 |1.27.2 |
|grpcio-tools    |1.25.0 |1.27.2 |
|pika            |0.12.0 |1.1.0 (REMOVED)|
|protobuf        |3.7.0  |3.11.3 |
|python-pkcs11   |0.5.0  |0.6.0 (REMOVED)|
|requests        |2.20.0 |2.23.0 |
|sanic           |18.12.0|19.12.2 (REMOVED)|
|websockets      |6.0    |8.1    |
|yappi           |0.98   |1.2.3  |

> Note that `dataclasses` is new from python 3.7. So it is removed from requirements.

### CODE REVIEW

**Review by windies21 — APPROVED** [review]

File: requirements.txt [hunk 1]
```diff
@@ -1,25 +1,21 @@
...
 coincurve==13.0.0
 setproctitle==1.1.10
 transitions==0.6.8
 verboselogs==1.7
-websockets==6.0
-yappi==0.98
+websockets==8.1
+yappi==1.2.3
 eth-keyfile==0.5.1
 dataclasses; python_version < '3.7'
```
Comment (yakkle) [inline requirements.txt]: This requirement is no more need.

Comment (metalg0su) [reply requirements.txt]: Your review intends that we "remove" those dependencies, not to "keep" current version of them. Right?

> About websockets
I tried to remove `websockets` at first.
But there's a explicit usage of it in Loopchain, and it is installed as a dependency of dependency.

> About yappi
I think removal of `yappi` will raise error if someone runs Loopchain by `$ loop [...]`. (See `./loopchain/__main__.py`

------
Unless we fix some code lines, I think removal of those dependencies could raise Exception.

Comment (yakkle) [reply requirements.txt]: I mean `dataclasses; python_version < '3.7'`.

Comment (metalg0su) [reply requirements.txt]: Oh, apology.

[dataclasses](https://docs.python.org/3.7/library/dataclasses.html) is New in version 3.7!

I see. 

Thank you.

Comment (metalg0su) [reply requirements.txt]: But I found that python version under 3.7 is still able to install Loopchain.
I think we must enforce python version by using `python_requires` in `setup.py` or whatever.

Comment (yakkle) [reply requirements.txt]: loopchain support 3.7.x only in [README](https://github.com/icon-project/loopchain#requirements).

Comment (metalg0su) [reply requirements.txt]: README says do not install python<3.7 but, I mean basically it is not prohibited.

**Review by yakkle — APPROVED** [review]

**Review by yakkle — APPROVED** [review]

**Review by windies21 — APPROVED** [review]

**Pull request merged**